csvcubed/csvcubed/readers/cubeconfig/v1_0/columnschema.py

```diff
@@ -334,7 +334,7 @@ def map_to_qb_observation(self) -> QbObservationValue:
             if isinstance(self.measure, str):
                 measure = ExistingQbMeasure(self.measure)
 
-            elif isinstance(self.measure, NewMeasures):
+            elif isinstance(self.measure, Measure):
```

Comment:
Good catch. We probably need another ticket to go back and ensure we've got enough test coverage in this area.

Reply:
Issue raised https://app.zenhub.com/workspaces/features-squad-61b72275ac896c0010a1b00b/issues/gss-cogs/csvcubed/427.